views/default/forms/widgets/save.php

```diff
@@ -2,12 +2,12 @@
 /**
  * Elgg widget edit settings
  *
- * @uses $vars['widget']
+ * @uses $vars['entity']
```

Comment:
this breaks our backwards compatibility with regards to view vars http://learn.elgg.org/en/stable/appendix/releases.html#views